TestRunner.py

```diff
@@ -28,30 +29,43 @@ def load_test_cases(test_json):
 
 
 class TransformationRuns(object):
-    def __init__(self, interface, perturbation_type, load_tests=True):
+    def __init__(self, interface, name_of_transformation, load_tests=True):
```

Comment:
Would it be more convenient to use `transformation_name` instead?

Reply:
Done..